packages/build-info/src/frameworks/observable.ts

```diff
@@ -22,8 +22,4 @@ export class Observable extends BaseFramework implements Framework {
     light: '/logos/observable/default.svg',
     dark: '/logos/observable/default.svg',
   }
-
-  env = {
-    NODE_VERSION: '20',
```

Comment:
No requirement for 20 specifically, supports > 18
https://github.com/observablehq/framework/blob/main/package.json#L139-L141